docs/UserGuide.md

```diff
@@ -186,12 +193,11 @@ _Details coming soon ..._
 
 ## Command summary
 
-Action | Format, Examples
---------|------------------
-**Add** | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 t/friend t/colleague`
-**Clear** | `clear`
-**Delete** | `delete INDEX`<br> e.g., `delete 3`
-**Edit** | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`
-**Find** | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`
-**List** | `list`
-**Help** | `help`
+| Action             | Format, Examples                                                                                 |
+|--------------------|--------------------------------------------------------------------------------------------------|
+| **Add**            | `add n/NAME p/PHONE_NUMBER e/EMAIL`<br> e.g., `add n/John Doe p/98928479 e/johndpiano@xmail.com` |
+| **Delete**         | `delete n/NAME`<br> e.g., `delete n/Betsy Crowe`                                                 |
+| **Tag Instrument** | `tag instrument n/NAME i/TAG…​`<br> e.g.,`tag instrument n/John Doe i/piano i/guitar`            |
+| **Tag Genre**      | `tag genre n/NAME t/TAG…​`<br> e.g., `tag genre n/John Doe t/pop t/rock`                         |
```

Comment:
Would using the index as a reference (just like how AB3 has done it), like `delete INDEX` instead of using the name (as multiple musicians may have the same name) more appropriate? Same applies to `tag instrument` and `tag genre`.

Reply:
Actually, it is OK to leave it as it is now. We can make individual changes to this part after we define the command for our responsible feature!

Reply:
I've edited this section to reflect the commands described in all the PRs so far. Thanks for the suggestion!